### ISSUE TITLE
About CP column

### ISSUE THREAD

**Issue by munkh-erdene** [SwaksharDeb/stock-trader#4]
First of all thank you for sharing your code,

Can you tell me which indicator is this CP column?

**Comment by SwaksharDeb** [comment]
'CP' stands for fifteen days moving average. Currently, I am working on
this code. Feel free to contact me if you find any bug

**Comment by munkh-erdene** [comment]
Thank you,

I have a two issue.

1. When i feed large data its learning tooooooooo slow. I'm sorry im newbie in RL. Do you think is that enough to learn to trade 250 rows data?
2. Pytorch is allocated only 4-6 % of GPU (I know its not about your code), Do you know how to extend?

**Comment by SwaksharDeb** [comment]
It's because of the learning rate scheduler. Change the UPDATE_EVERY_LR to
the size of your data.
About your second question, I am not sure about the answer.
